input/core/src/commonMain/kotlin/majestic/calendar/tools/DatePickerState.kt

```diff
@@ -0,0 +1,42 @@
+package majestic.calendar.tools
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import kotlinx.datetime.LocalDate
+
+@Stable
+class DatePickerState(
+    initialDate: LocalDate? = null,
+    initialMonth: LocalDate
```

Comment:
Why do you have a different `initialDate` and `initialMonth` if someone can pass `initialiDate = LocalDate(1, June, 2025)`?